userWhitelist.json

```diff
@@ -22,6 +22,7 @@
     "aks681",
     "srijanreddy98"
   ],
+  "codeOwnerReviewer":"DubeySandeep",
```

Comment:
Why we need constant? Why don't we retrieve this from the CODEOWNERS file?

Reply:
We decided to go this way because we felt it was better to keep it constant as this usually doesn't change for a long period of time. Also, it would be easier to reference this for future purposes too. @vojtechjelinek

Reply:
Ok.